packages/docs/src/styles/_sections.scss

```diff
@@ -69,6 +69,7 @@
 }
 
 #{section-name("Labels")},
+#{section-name("Helper Text")},
```

Comment:
Totally right! Then I guess your `margin` solution sounds good then. Instead of `$pt-grid-size * 0.7`, would it be possible to compute it? Something like `($pt-input-height - $label-font-size) / 2`? Same for large? @giladgray will know better